test/annex/layer/sequence_test.exs

```diff
@@ -101,7 +103,13 @@ defmodule Annex.Layer.SequenceTest do
       assert_in_order(seq2)
       assert {%Sequence{} = seq3, _} = Sequence.feedforward(seq2, [1.0])
       assert_in_order(seq3)
-      props = Backprop.new(cost_gradient: 0.03, net_loss: 0.1, loss_pds: [0.4])
+
+      props =
+        Backprop.new(
+          negative_gradient: 0.1
+          # derivative: &Activation.sigmoid_deriv/1
```

Comment:
remove this commented code. `:derivative` is not needed. This issue was fixed by the `rem(n, 3) == 0` clause for generating layers above.